### PR TITLE
Fix UI tests

### PR DESCRIPTION
**Pull request by charliescheer**
### Fix
The UI Tests are failing currently so the CI is unhappy.  We can't have that, so I have gone ahead and fixed the failures.

Note that I am currently getting different failures locally as compared to what is happening on the CI.  What this PR fixes is the UI Tests and how they run locally.  If the CI fails this PR I will phone an infrastructure friend to take a look and see what if we can figure out why.

### Test
***(Required)*** List the steps to test the behavior.  For example:
1.  Select the UI test target and run all the tests.  
Confirm they all pass.

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### CODE REVIEW

File: SimplenoteUITests/Generic.swift [hunk 1]
```diff
@@ -154,8 +154,10 @@ class WebView {
 
     class func tapDone() {
         let doneButton = app.buttons[UID.Button.done]
-        guard doneButton.exists else { return }
-        doneButton.tap()
+        guard doneButton.exists else {
+            return
+        }
+        doneButton.tapCenterCoordinates(in: app)
```
Comment (charliescheer) [inline SimplenoteUITests/Generic.swift:160]: For some reason the webkit done button couldn't be tapped on by the test runner.  So getting the center point and tapping the coords instead

File: SimplenoteUITests/NoteList.swift [hunk 2]
```diff
@@ -32,11 +32,11 @@ class NoteList {
     }
 
     class func getNoteCellHeight(_ noteName: String) -> CGFloat {
-        return NoteList.getNoteCell(noteName).frame.height
+        return NoteList.getNoteCell(noteName).frame.height.rounded()
     }
 
     class func openAllNotes() {
-        guard !NoteList.isAllNotesListOpen() else { return }
+        guard !isAllNotesListOpen() else { return }
```
Comment (charliescheer) [inline SimplenoteUITests/NoteList.swift:39]: Linter warning

File: SimplenoteUITests/NoteList.swift [hunk 3]
```diff
@@ -32,11 +32,11 @@ class NoteList {
     }
 
     class func getNoteCellHeight(_ noteName: String) -> CGFloat {
-        return NoteList.getNoteCell(noteName).frame.height
+        return NoteList.getNoteCell(noteName).frame.height.rounded()
```
Comment (charliescheer) [inline SimplenoteUITests/NoteList.swift:35]: This was returning `80.99999999999997` consistently.  But we are looking for `81`.  close enough...

File: SimplenoteUITests/SimplenoteUITestsNoteEditor.swift [hunk 4]
```diff
@@ -114,7 +114,7 @@ class SimplenoteUISmokeTestsNoteEditor: XCTestCase {
 
         trackStep()
         NoteEditor.swipeToPreview()
-        PreviewAssert.staticTextWithExactValueShownOnce(value: noteNameInitial)
+        PreviewAssert.staticTextWithExactLabelShownOnce(label: noteNameInitial)
```
Comment (charliescheer) [inline SimplenoteUITests/SimplenoteUITestsNoteEditor.swift:117]: When the checkbox was added the value was not discoverable but the label was.

**Review by jleandroperez — APPROVED** [review]
Niiiiiceee 🚀 !!!

**Pull request merged**